commbench crashes in the ctxt test on netlrts-linux-x86_64-clang

This failure was seen in autobuild: http://charm.cs.illinois.edu/autobuild/old.2020_04_23__01_02/netlrts-linux-x86_64-clang.txt

```
make[3]: Entering directory '/scratch/autobuild/netlrts-linux-x86_86-clang/charm/netlrts-linux-x86_64-clang/benchmarks/converse/commbench'
../../../bin/charmc -optimize -production  -optimize -production  memoryAccess.c
../../../bin/charmc -optimize -production  -optimize -production  commbench.c
../../../bin/charmc -optimize -production  -optimize -production  overhead.c
../../../bin/charmc -optimize -production  -optimize -production  timer.c
../../../bin/charmc -optimize -production  -optimize -production  proc.c
../../../bin/charmc -optimize -production  -optimize -production  smputil.c
../../../bin/charmc -optimize -production  -optimize -production  pingpong.c
../../../bin/charmc -optimize -production  -optimize -production  flood.c
../../../bin/charmc -optimize -production  -optimize -production  broadcast.c
../../../bin/charmc -optimize -production  -optimize -production  reduction.c
../../../bin/charmc -optimize -production  -optimize -production  ctxt.c
../../../bin/charmc -optimize -production  -optimize -production  pingpong-cachemiss.c
../../../bin/charmc -optimize -production  -optimize -production  -o pgm memoryAccess.o commbench.o overhead.o timer.o proc.o smputil.o pingpong.o flood.o broadcast.o reduction.o ctxt.o pingpong-cachemiss.o -language converse++
../../../bin/testrun  ./pgm +p4  ++local
Charmrun> scalable start enabled. 
Charmrun> started all node programs in 0.027 seconds.
Charm++> Running in non-SMP mode: 4 processes (PEs)
Converse/Charm++ Commit ID: f76a4fb
Isomalloc> Synchronized global address space.
Charm++> scheduler running in netpoll mode.
proc module init
proc module init
proc module init
proc module init
Charm++> Running on 1 hosts (1 sockets x 4 cores x 2 PUs = 8-way SMP)
Charm++> cpu topology info is gathered in 0.032 seconds.
...
...
...
[broadcast] (CmiReduce) 2.706548e-03 seconds per 4 bytes
[broadcast] (CmiReduce) 3.912032e-03 seconds per 16 bytes
[broadcast] (CmiReduce) 3.791792e-03 seconds per 64 bytes
[broadcast] (CmiReduce) 3.507205e-03 seconds per 256 bytes
[broadcast] (CmiReduce) 5.090213e-03 seconds per 1024 bytes
[broadcast] (CmiReduce) 9.981157e-03 seconds per 4096 bytes
[broadcast] (CmiReduce) 3.016170e-03 seconds per 16384 bytes
[broadcast] (CmiReduce) 1.182148e-02 seconds per 65536 bytes
[broadcast] (CmiReduce) 2.210896e-02 seconds per 262144 bytes
[broadcast] (CmiReduce) 5.062893e-02 seconds per 1048576 bytes
[ctxt] initiated
[ctxt] Thread Context Switching Overhead = 3.624036e-06 seconds
------------- Processor 0 Exiting: Called CmiAbort ------------
Reason: Msg handler does not exist, possible race condition during init

[0] Stack Traceback:
  [0:0] pgm 0x43e3a2 
  [0:1] pgm 0x423ddb 
  [0:2] pgm 0x423b92 CsdScheduler
  [0:3] pgm 0x4404de ConverseInit
  [0:4] pgm 0x41d540 main
  [0:5] libc.so.6 0x7ffff6d6e830 __libc_start_main
  [0:6] pgm 0x41c749 _start
Fatal error on PE 0> Msg handler does not exist, possible race condition during init

Makefile:13: recipe for target 'test' failed
make[3]: Leaving directory '/scratch/autobuild/netlrts-linux-x86_86-clang/charm/netlrts-linux-x86_64-clang/benchmarks/converse/commbench'
Makefile:21: recipe for target 'test-commbench' failed
make -C cthtest test OPTS='-optimize -production  -optimize -production ' TESTOPTS='++local'

real	4m15.376s
user	0m0.019s
sys	0m0.000s
make[3]: *** [test] Error 1
make[2]: *** [test-commbench] Error 2
```

This happened on UCX too:

```
make[3]: Entering directory `/scratch/users/nbhat4/autobuild/ucx/charm/ucx-linux-x86_64/benchmarks/converse/commbench'
../../../bin/testrun  ./pgm +p4  

Running on 4 processors:  ./pgm 
./charmrun: line 152: mvapich2-start-mpd: command not found
Charm++> Running in non-SMP mode: 4 processes (PEs)
Converse/Charm++ Commit ID: e3a2c6d
Charm++ built with internal error checking enabled.
Do not use for performance benchmarking (build without --enable-error-checking to do so).
Isomalloc> Synchronized global address space.
proc module initproc module init
proc module init

proc module init
Charm++> Running on 2 hosts (1 sockets x 3 cores x 1 PUs = 3-way SMP)
Charm++> cpu topology info is gathered in 0.002 seconds.
[broadcast] initiated
[broadcast] (Consecutive CmiSyncBroadcastAllAndFree) 6.425332e-07 seconds per 4 bytes
[broadcast] (Consecutive CmiSyncBroadcastAllAndFree) 6.735703e-07 seconds per 16 bytes
[broadcast] (Consecutive CmiSyncBroadcastAllAndFree) 8.018340e-07 seconds per 64 bytes
[broadcast] (Consecutive CmiSyncBroadcastAllAndFree) 1.094582e-06 seconds per 256 bytes
[broadcast] (Consecutive CmiSyncBroadcastAllAndFree) 1.309877e-06 seconds per 1024 bytes
[broadcast] (Consecutive CmiSyncBroadcastAllAndFree) 2.036258e-06 seconds per 4096 bytes
[broadcast] (Consecutive CmiSyncBroadcastAllAndFree) 6.046979e-06 seconds per 16384 bytes
[broadcast] (Consecutive CmiSyncBroadcastAllAndFree) 2.099982e-05 seconds per 65536 bytes
[broadcast] (Consecutive CmiSyncBroadcastAllAndFree) 9.356099e-05 seconds per 262144 bytes
[broadcast] (Consecutive CmiSyncBroadcastAllAndFree) 3.792557e-04 seconds per 1048576 bytes
[broadcast] (CmiSyncBroadcastAllAndFree) 1.580113e-06 seconds per 4 bytes
[broadcast] (CmiSyncBroadcastAllAndFree) 1.627361e-06 seconds per 16 bytes
[broadcast] (CmiSyncBroadcastAllAndFree) 2.162432e-06 seconds per 64 bytes
[broadcast] (CmiSyncBroadcastAllAndFree) 2.542303e-06 seconds per 256 bytes
[broadcast] (CmiSyncBroadcastAllAndFree) 3.320748e-06 seconds per 1024 bytes
[broadcast] (CmiSyncBroadcastAllAndFree) 5.224684e-06 seconds per 4096 bytes
[broadcast] (CmiSyncBroadcastAllAndFree) 1.322270e-05 seconds per 16384 bytes
[broadcast] (CmiSyncBroadcastAllAndFree) 3.226037e-05 seconds per 65536 bytes
[broadcast] (CmiSyncBroadcastAllAndFree) 1.404162e-04 seconds per 262144 bytes
[broadcast] (CmiSyncBroadcastAllAndFree) 5.378389e-04 seconds per 1048576 bytes
[reduction] initiated
[broadcast] (CmiReduce) 3.716635e-06 seconds per 4 bytes
[broadcast] (CmiReduce) 3.728117e-06 seconds per 16 bytes
[broadcast] (CmiReduce) 4.248643e-06 seconds per 64 bytes
[broadcast] (CmiReduce) 4.437857e-06 seconds per 256 bytes
[broadcast] (CmiReduce) 4.887824e-06 seconds per 1024 bytes
[broadcast] (CmiReduce) 6.169404e-06 seconds per 4096 bytes
[broadcast] (CmiReduce) 9.423370e-06 seconds per 16384 bytes
[broadcast] (CmiReduce) 2.696743e-05 seconds per 65536 bytes
[broadcast] (CmiReduce) 1.002547e-04 seconds per 262144 bytes
[broadcast] (CmiReduce) 3.611678e-04 seconds per 1048576 bytes
[ctxt] initiated
[ctxt] Thread Context Switching Overhead = 1.763191e-07 seconds
------------- Processor 0 Exiting: Called CmiAbort ------------
Reason: Converse zero handler executed-- was a message corrupted?

[0] Stack Traceback:
  [0:0] pgm 0x4441e5 CmiAbortHelper(char const*, char const*, char const*, int, int)
  [0:1] pgm 0x4442ed 
  [0:2] pgm 0x425090 
  [0:3] pgm 0x425ec5 CsdScheduleForever
  [0:4] pgm 0x42616d CsdScheduler
  [0:5] pgm 0x44962a ConverseInit
  [0:6] pgm 0x41eece main
  [0:7] libc.so.6 0x2ab4e7c57555 __libc_start_main
  [0:8] pgm 0x41db6f 
[mpiexec@golub291] HYDU_sock_write (utils/sock/sock.c:294): write error (Bad file descriptor)
[mpiexec@golub291] HYD_pmcd_pmiserv_send_signal (pm/pmiserv/pmiserv_cb.c:177): unable to write data to proxy
[mpiexec@golub291] ui_cmd_cb (pm/pmiserv/pmiserv_pmci.c:79): unable to send signal downstream
[mpiexec@golub291] HYDT_dmxu_poll_wait_for_event (tools/demux/demux_poll.c:76): callback returned error status
[mpiexec@golub291] HYD_pmci_wait_for_completion (pm/pmiserv/pmiserv_pmci.c:198): error waiting for event
[mpiexec@golub291] main (ui/mpich/mpiexec.c:340): process manager error waiting for completion
make[3]: *** [test] Terminated
```

--- a/benchmarks/converse/commbench/ctxt.c
+++ b/benchmarks/converse/commbench/ctxt.c
@@ -7,6 +7,7 @@ void ctxt_init(void) {
   double starttime, endtime;
   int i;
   EmptyMsg msg;
+  CmiInitMsgHeader(msg.core, sizeof(EmptyMsg));
 
   starttime = CmiWallTimer();
   for (i = 0; i < NITER; i++) CthYield();

